app/controllers/submit.js

```diff
@@ -1339,11 +1459,20 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
     },
 
     _moveFromBasics() {
-        this.send('next', this.get('_names.2'));
+        this.send('next', 'Basics');
+    },
+
+    _moveFromCoi() {
+        this.send('next', 'COI');
+    },
+
+    _failMoveFromCoi() {
+        // If model save fails, do not transition, save original vales
+        this.get('toast').error(this.get('i18n').t('submit.coi_error'));
     },
 
     _moveFromSupplemental() {
-        this.send('next', this.get('_names.5'));
+        this.send('next', 'Supplemental');
```

Comment:
A part of me wants to put these into an Enum, but another part of me says that's not necessary... 🤷‍♂

Reply:
Not a bad idea. We should probably do that when we add other sections.